jekyll/_cci2/env-vars.md

```diff
@@ -16,6 +16,10 @@ This document describes using environment variables in CircleCI in the following
 
 To add keys or secret environment variables to your private project, use the Environment Variables page of the Build > Project > Settings in the CircleCI application. The value of the variables are neither readable nor editable in the app after they are set. To change the value of an environment variable, delete the current variable and add it again with the new value. It is possible to add individual variables or to import variables from another project. Do **not** add keys or secrets to a public CircleCI project.
 
+### Adding Global Environment Variables
+
+To add global environment variables that may be shared across projects, use the Settings > Contexts page of the CircleCI application. See the [Contexts]( {{ site.baseurl }}/2.0/contexts/) documentation for instructions.
```

Comment:
same as above. am i missing something? you can get to org settings regardless of what UI you are using.